handler/src/main/java/io/netty/handler/ssl/OpenSslSessionContext.java

```diff
@@ -35,13 +35,15 @@
 
     private final OpenSslSessionStats stats;
     final ReferenceCountedOpenSslContext context;
+    private final OpenSslKeyMaterialProvider provider;
 
     // IMPORTANT: We take the OpenSslContext and not just the long (which points the native instance) to prevent
     //            the GC to collect OpenSslContext as this would also free the pointer and so could result in a
     //            segfault when the user calls any of the methods here that try to pass the pointer down to the native
     //            level.
-    OpenSslSessionContext(ReferenceCountedOpenSslContext context) {
+    OpenSslSessionContext(ReferenceCountedOpenSslContext context, OpenSslKeyMaterialProvider provider) {
         this.context = context;
+        this.provider = provider;
```

Comment:
Since this variable is private and not used (other than destruction), it would seem a good idea to have a comment mention what is actually using this object elsewhere and why that corresponds to this object's lifetime.